### ISSUE TITLE
Remove flagger?

### ISSUE THREAD

**Issue by zeckalpha** [randsleadershipslack/destalinator#137]
It's underutilized and a frequent source of bugs. Can we get rid of it?

**Comment by TheConnMan** [comment]
Flagging based on reactions feels like a more real-time operation, not nightly. I swear I've seen a bot out there that does what the flagger does in real-time (though not based on chatted in rules). Might make sense to either use one of those bots or build a different bot using the Slack Events API.

**Comment by kmarekspartz** [comment]
Yeah https://reacji-channeler.builtbyslack.com is the real-time one.

Some people prefer the async reminder as a spaced repetition sort of thing. Additionally, the flagger allows arbitrary thresholds, e.g. requiring two or more reacji before notifying.

In the Rands Slack, I have converted the single reacji cases to use reacji channeler, but a few rules still remain.

**Comment by TheConnMan** [comment]
That's the one. Makes sense, especially for rules which take multiple reactions.
